website/docs/faqs/Core/install-pip-os-prereqs.md

```diff
@@ -33,7 +33,7 @@ python --version
 
 ```
 
-If you need a compatible version, you can download and install [Python version 3.8 or higher for MacOS](https://www.python.org/downloads/macos).
+If you need a compatible version, you can download and install [Python version 3.9 or higher for MacOS](https://www.python.org/downloads/macos).
 
 If your machine runs on an Apple M1 architecture, we recommend that you install dbt via [Rosetta](https://support.apple.com/en-us/HT211861). This is necessary for certain dependencies that are only supported on Intel processors.
```

Comment:
@matthewshaver do we also want to update this section for windows: https://docs.getdbt.com/faqs/Core/install-pip-os-prereqs.md#windows